go.mod

```diff
@@ -41,3 +38,7 @@ require (
 	sigs.k8s.io/controller-runtime v0.8.3
 	sigs.k8s.io/yaml v1.2.0
 )
+
+// TODO(sertac): replace this before merging
```

Comment:
Fix TODO before merging

Reply:
frameworks pr is merged and vendored in now